### DISCUSSION TITLE
잘 밤에 쓸데없는 생각하기/Adieu 2022, Bienvenue 2023

### DISCUSSION THREAD

**Discussion by giscus** [ujuc/ujuc.github.io#69] [General]
# 잘 밤에 쓸데없는 생각하기/Adieu 2022, Bienvenue 2023

올해도 지나갔다.

https://ujuc.github.io/2022/12/29/adieu-2022-bienvenue-2023/

**Comment by hyuntoe** [comment]
고생 많으셨습니다
새해 복 많이 받으시고 건강하십쇼~

**Comment by hyuntoe** [comment]
고생 많으셨습니다
새해 복 많이 받으시고 *건강*하십쇼~
